omniport/core/guest_auth/utils/create_guest.py

```diff
@@ -0,0 +1,27 @@
+import swapper
+import random
+import string
+from datetime import date
```

Comment:
Always, always, always
```suggestion
import datetime
```

Reply:
That way when in the code you refer to datetime.datetime or datetime.date you can easily understand whether your refering to the date-time inside the 'date & time' package or just the date inside the 'date & time' package.